src/mdio/creators/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""MDIO Data creation API."""
+
+from mdio.creators.mdio import create_empty_mdio
```

Comment:
```suggestion
from mdio.creation.mdio import create_empty_mdio
```

Thoughts on `creation` over `creators` for module name?